docs/change-log/index.md

```diff
@@ -13,16 +14,181 @@
+
+#### UI
+- The new **Projects** home page provides easy and intuitive access to the full project lifecycle in three phases, with links to the 
+relevant wizards under each phase heading: ingesting and processing data, developing and training a model, deploying and monitoring the project.
+
+<p align="center"><img src="../_static/images/project-homepage.png" alt="mlrun-project-homepage" /></p><br>
+
+<a id="api-130"></a>
+### Deprecated and removed APIs
+Starting with v1.3.0, and continuing in subsequent releases, obsolete functions are getting removed from the code.
+
+**Deprecated and removed from v1.3.0 code**<br>
+The following MLRun APIs have been deprecated since at least v1.0.0 and are now removed from the code:
+
+| Deprecated/removed                   | Use instead                                   |
+| ------------------------------------ | --------------------------------------------- |
+| `project.workflows`                  | `project.spec.workflows`, such as `get_or_create_project`, `load_project` |
+| `project.functions`                  | `project.spec.functions`                      |
+| `project.artifacts`                  | `project.spec.artifacts`                       |
+| `project.func()`                     | `project.get_function()`                       |
+| `project.create_vault_secrets()`     | `project.set_secrets()`                        |
+| `project.get_vault_secret()`         | `proj.with_secrets`                            |
+| `MlrunProjectLegacy` class           | `MlrunProject`                                 |
+| Feature-store: usage of state in graph. For example: `add_writer_state`, and the `after_state` parameter in `_init_` methods.  | `step`                        |
+| `mount_path` parameter in mount_v3io() | `volume_mounts`                        |
+| `NewTask`                            | `run_function()`                        |
+
+
+
+**Deprecated APIs, will be removed in v1.5.0**<br>
+These APIs will be removed from the v1.5.0 code. A FutureWarning appears if you try to use them in v1.3.0.
+| Deprecated / to be removed                       | Use instead                                   |
+| ------------------------------------------------ | --------------------------------------------- |
+| project-related parameters of `set_environment`. (Global-related parameters will not be deprecated.) | Project APIs such as `get_or_create_project`, `load_project` |
+| `AbstractSparkJobSpec.gpus`, `KubeResource.gpus` | `with_limits`                 |
+| Dask `gpus`                                      | `with_scheduler_limits` / `with_worker_limits`   |
+| Dask `with_limits`                               | `with_scheduler_limits` / `with_worker_limits`    |
+| Dask `with_requests`                             | `with_scheduler_requests` / `with_worker_requests`    |
+| `Job gpus`                                       | `with_limits`                       |
```

Comment:
`Job gpus` is exactly the same as `KubeResource.gpus`, so no need to have it twice.